web/src/test/java/com/microsoft/applicationinsights/web/internal/httputils/HttpServerHandlerTest.java

```diff
@@ -161,7 +157,11 @@ public void timeStatusCodeAreSetWhenHandleEndIsInvoked() throws MalformedURLExce
         assertThat(rt.getUrl().toString(), equalTo(url));
         assertThat(rt.getContext().getUser().getUserAgent(), equalTo("User-Agent"));
         assertThat(rt.getTimestamp(), is(CoreMatchers.<Date>notNullValue()));
-
+        try {
+            TimeUnit.MILLISECONDS.sleep(100); // pause for a moment
```

Comment:
Can you add comment why `sleep` is needed?

Reply:
will do